experiments/README.md

```diff
@@ -23,24 +23,45 @@ this is the build tool we use to run tests, etc.
 Install library dependencies (it is optional, but recommended to install these
 in a [Virtual Environment](https://docs.python.org/3/tutorial/venv.html):
 
-    ```shell
-    # The python3 way to create and use virtual environment
-    # (optional, but recommended):
-    python3 -m venv .pyenv
-    source .pyenv/bin/activate
-    # Install dependencies
-    pip install -r requirements.txt
+```shell
+# The python3 way to create and use virtual environment
+# (optional, but recommended):
+python3 -m venv .pyenv
+source .pyenv/bin/activate
+# Install dependencies
+pip install -r requirements.txt
 
-    # ... do stuff ...
+# ... do stuff ...
 
-    # Exit your virtual environment.
-    deactivate
-    ```
+# Exit your virtual environment.
+deactivate
+```
 
 ### Cloud and ML Engine configuration
 
-TODO(nthain)
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+2. Log in: 
+```shell
+gcloud auth application-default login
+```
+You will be promoted to visit a page in the browser; follow the login instructions there.
+
+3. Set the project:
+```shell
+gcloud config set project wikidetox
```

Comment:
Let's replace 'wikidetox' here with '[PROJECT]'. That way, the instructions are general to people outside of the team currently working on it.